Formula/glibc.rb

```diff
@@ -95,43 +95,166 @@ class Glibc < Formula
 
   depends_on "binutils" => :build
   depends_on GawkRequirement => :build
-  depends_on "linux-headers@4.4" => :build
+  depends_on "linux-headers@5.15" => :build
   depends_on MakeRequirement => :build
   depends_on SedRequirement => :build
   depends_on BrewedGlibcNotOlderRequirement
   depends_on :linux
   depends_on LinuxKernelRequirement
 
-  # GCC 4.7 or later is required.
-  fails_with gcc: "4.6"
+  # TODO: build bzip2 with glibc@2.13
+  uses_from_macos "bzip2" => :build
```

Comment:
Rebuilding `bzip2` with an ancient `glibc` for a throwaway compiler which you may not always need seems a bit excessive.

Reply:
I think it's useful to do this in general though because `bzip2` is sometimes needed for extracting resources in formulae and `brew` conveniently installs it for us in that case.  It's also extremely simple now that we set up the right flags in `brew`, and there's no real downside to doing this for a formula as simple as `bzip2`.

Reply:
Actually I have a simple workaround - `brew` will automatically install `bzip2` because it detects resources with `.bz2` extensions.  However because we put `bootstrap/bin` first in the PATH, we can still install `bzip2` there if building `gcc` and it gets used instead.  The only downside is that I can't use `which` to detect `bzip2` anymore because it shows up in the PATH in `HOMEBREW_PREFIX/bin`, so instead we have to check if `/usr/bin/bzip2` exists.  I'm happy to use this workaround for now to get this merged and we can iterate on this later if needed.